src/explorer/trialApp/TrialAppTreeItem.ts

```diff
@@ -51,6 +51,14 @@ export class TrialAppTreeItem extends SiteTreeItemBase implements ISiteTreeItem
         this._tutorialNode = new GenericTreeItem(this, { label: 'Show tutorial', commandId: 'appService.ShowTutorial', contextValue: 'showTutorial', iconPath: getThemedIconPath('book') });
         this.logFilesNode = new LogFilesTreeItem(this, this.client);
         this.deploymentsNode = new TrialAppDeploymentsTreeItem(this, this.client, {}, {});
+
+        // minutes * seconds * ms
+        const interval: number = 5 * 60 * 1000;
```

Comment:
I'd rather you use setTimeout rather than setInterval.  You don't ever actually clear this out, so I'm assuming that this interval would still exist even after you remove the tree item.

With set timeout, you can check if the tree item still exists before calling another setTimeout.  Also, I'm okay with the interval being every 60 seconds since I'd like to see it tick down every minute.